### PR TITLE
Changes to development

### PR DESCRIPTION
**Pull request by Abner21**

### CODE REVIEW

File: GradientsProject/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <li class="firstChild"><a href="#" class="inline-block"><i class="fab fa-apple apple-button"></i></a></li> <!--
+        --><li><a href="#">Store</a></li><!--
+         --><li><a href="#">Mac</a></li><!--
+        --><li><a href="#">iPod</a></li><!--
+        --><li><a href="#">iPhone</a></li><!--
+        --><li><a href="#">iPad</a></li><!--
+        --><li><a href="#">iTunes</a></li><!--
+        --><li><a href="#">Support</a></li><!--
+
+          -->
```
Comment (samgaco) [inline GradientsProject/index.html]: You should avoid having this --> arrows within the code.

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @Abner21 :vulcan_salute: 

Great job so far :clap: It is looking good :100:  here are some changes to be made:

* The **elements in the** **navigation bar** should be buttons and be arranged with either floats or display flex.
* The search `<input>` tag should have the[ focus effect](https://www.w3schools.com/cssref/sel_focus.asp) enabled in the CSS file.
*  The _firstIphone_, _secondIphone_, etc. should be within the footer and those '<div>' tags should be wrapped within `<a>` tags so they are clickable.
* Add project description in the **read me file**, is also suggested to add your contact and a live-demo of the site for example. [Here ](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)you have an example template of a readme file.
* The main content (from the navigation bar until the footer) should be wrapped in a `<main>` tag, and the background of this tag should be the main image of the site.
* Is recommended to have the lists in the footer arranged with floats or display flex.

### That's it. Please :pray: work on these changes and submit back for another code review when you are done. Good spirit and happy coding! :muscle:

**Comment by mosaaleb** [comment]
Reviewing

File: index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    </a>
+
+    <a href="index.html">
+    <div class="everyoneHas">
+
+    </div>
+    </a>
+
+    <div class="watchStories">
+      <a class="container" href="index.html">Watch the "Your Verse" film and explore the stories <i class="far fa-play-circle"></i></a>
```
Comment (mosaaleb) [inline index.html]: There text 'watch the ....' should be image.
- [ ] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [ ] The `<div>` tag  inside the `<a>` tag should use an image as background -- [ ] The text on the original page is actually an image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+        <a href="index.html" class="inline-block"><i class="fab fa-apple apple-button"></i></a>
+        <a href="index.html" class="inline-block">Store</a>
+         <a href="index.html" class="inline-block">Mac</a>
+        <a href="index.html" class="inline-block">iPod</a>
+        <a href="index.html" class="inline-block">iPhone</a>
+        <a href="index.html" class="inline-block">iPad</a>
+        <a href="index.html" class="inline-block">iTunes</a>
+        <a href="index.html" class="inline-block">Support</a>
+      <div class="lastItem">
```
Comment (mosaaleb) [inline index.html]: [optional]: You can use css `last-child` selector instead of defining a new `lastItem` class.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @Abner21 

#### Great job :clap:. I liked that you took care of different browser vendors by using vendor prefixes. The UI is clean. You have met most of the requirements, yet there is minor requirements needs to be done before moving on. See my comments below.


Also, 

### General:

- [ ] The project should have a descriptive README.md
- [ ] Let's make sure that we have good style guidelines by using [stickler](https://github.com/microverseinc/linters-config/tree/master/css).

Please submit a new code review once these requirements are met.

#### Happy coding :computer:

**Comment by samgaco** [comment]
### Hi @Abner21 :hand: 

### The project looks perfect now and is approved :clap: 

**There is a problem though. The branch has a conflict, so you should solve that before merging to the master branch and submitting the completion form.**

Anyways, congratulations on finishing your first solo project. :muscle:

**Comment by Abner21** [comment]
Thanks for the valuable feedback, guys!

**Pull request merged**